sitestats/sitestats-impl-hib/src/java/org/sakaiproject/sitestats/impl/SitePresenceImpl.java

```diff
@@ -39,6 +39,7 @@ public class SitePresenceImpl implements SitePresence, Serializable {
 	private Date date;
 	private long duration;
 	private Date lastVisitStartTime;
+	private int currentOpenSessions;
```

Comment:
I would prefer using the object Integer vs a primitive as this is a db POJO where values can be NULL, even if you declare it not NULL in the HBM